docs/howto/content/python.md

```diff
@@ -39,3 +39,39 @@ os.getcwd()  # If successful:  "/drive/path/to/notebook"
   https://github.com/jupyterlite/jupyterlite/blob/main/examples/pyolite/emscripten-filesystem.ipynb
 [fs]: https://emscripten.org/docs/api_reference/Filesystem-API.html
 [caniuse-sw]: https://caniuse.com/serviceworkers
+
+## Fetching remote content
```

Comment:
While this is good, perhaps actually having a demo notebook (maybe showing seaborn) would be useful as well... and to ensure it _keeps_ working.

Reply:
Good point, thanks :+1: